### DISCUSSION TITLE
Self-hosted server for Revolt Servers

### DISCUSSION THREAD

**Discussion by sapphireenabler** [stoatchat/discussions#790] [Feature Suggestions]
I not able to login and even register due to Discord block in Russia because of deadbrain government. First i have to say thanks for open-source projects like yours. Second i have an idea to present. Is it possible to provide people self-hosted solution to unload revolt servers?

**Comment by IAmTomahawkx** [comment]
Anyone is welcome to create their own revolt instance, see https://github.com/revoltchat/self-hosted

**Reply by sapphireenabler** [reply]
Docker actually sounds like only option. Is it way to install directly on any linux (RHEL-based or Deb-based for example)?
